rollup-interface/src/state_machine/state.rs

```diff
@@ -0,0 +1,21 @@
+//! Trait that represents life time of the state
+//!
+
+/// Storage manager manager persistence and allows to work on state
+/// Temporal placeholder for ForkManager
+pub trait StateManager {
```

Comment:
Should it be `StorageManager` ?

Reply:
I think `Storage` is better, but the relevant docs will need to be updated as well.